pkgs/development/interpreters/python/hooks/default.nix

```diff
@@ -184,4 +184,12 @@ in rec {
       name = "python${python.pythonVersion}-sphinx-hook";
       deps = [ sphinx ];
     } ./sphinx-hook.sh) {};
+
+  unittestCheckHook = callPackage ({ }:
+    makeSetupHook {
+      name = "unittest-check-hook";
+      substitutions = {
+        inherit pythonCheckInterpreter;
+      };
+    } ./unittest-check-hook.sh) {};
```

Comment:
Please make sure to add this entry in the correct order. This file is generally ordered alphabetically.

Reply:
I think I did this because the Sphinx hook wasn't -- done.

Reply:
Yeah, another artifact.